src/PrestaShopBundle/DependencyInjection/PrestaShopExtension.php

```diff
@@ -41,14 +41,9 @@ class PrestaShopExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $configuration = new AddOnsConfiguration();
-        $config = $this->processConfiguration($configuration, $configs);
-
         $loader = new YamlFileLoader($container, new FileLocator(dirname(__DIR__) . '/Resources/config'));
         $env = $container->getParameter('kernel.environment');
         $loader->load('services_' . $env . '.yml');
-
-        $container->setParameter('prestashop.addons.categories', $config['addons']['categories']);
```

Comment:
I think it's a good idea to remove this parameter that was used only in one place However we should still note in the PR that it's a breaking change and detail under the description table that `prestashop.addons.categories` parameter has been removed (in case a module depended on it)

Reply:
Added this into the description of the PR and added the needs doc tag as well